src/enzyme_ad/jax/Dialect/EnzymeXLAOps.td

```diff
@@ -427,7 +427,11 @@ def SyrkOp: EnzymeXLA_Op<"blas.syrk", [Pure, SameOperandsAndResultElementType]>
   let summary = "Multiplication involving a symmetric matrix";
 
   let description = [{
-    C := alpha*A*A^T + beta*C, or C := alpha*A^T*A + beta*C, where alpha and beta are scalars. C must be a n x n symmetric matrix."
+    C := alpha*A*A^T + beta*C, or C := alpha*A^T*A + beta*C, where alpha and beta are
+    scalars. C must be a n x n symmetric matrix.
+
+    If `fill` is present, then both the upper and lower triangles of the matrix are filled.
+    Otherwise only accessing `uplo` is valid.
```

Comment:
rather we should describe this as accessing the non-uplo part is undefined behavior.

Reason being that if we want is_symmetric(syrk) == true always, then we need to leverage the fact that syrk(fill=false) could have filled the bottom with the symmetric part (and we chose the undefined behavior which best suited us). Otherwise fill = false being potentially defined means we cannot make that assumption